sqlite3-binding.h

```diff
@@ -1,4 +1,3 @@
-#ifndef USE_LIBSQLITE3
```

Comment:
This has to be manually added again, or it won't work properly for people using the libsqlite3 build tag. Likewise for sqlite3-binding.c. This is due to how cgo works. https://pkg.go.dev/cmd/cgo

> When the Go tool sees that one or more Go files use the special import "C", it will look for other non-Go files in the directory and compile them as part of the Go package.

Reply:
Oh thanks! I realized I should just use the upgrade tool. Which is very nice! Committed the result of running that tool.